src/test/java/org/jitsi/meet/test/base/AbstractBaseTest.java

```diff
@@ -0,0 +1,195 @@
+            testsToInclude
+                = getStringTokens(System.getProperty(TESTS_TO_INCLUDE_PNAME));
+        }
+
+        String thisClassName = this.getClass().getSimpleName();
+
+        // if test is explicitly in exclude, skip
+        if (testsToExclude.contains(thisClassName))
+        {
+            throw new SkipException("skips-" + thisClassName);
+        }
+
+        // if test is explicitly in include, run it
+        if (testsToInclude.contains(thisClassName))
+        {
+            return;
+        }
+
+        // if testsToRun is empty, then check if test implementation
+        // prefers to not be run by default
+        // or if testsToRun is not empty check whether it contains this test
+        if ((testsToRun.isEmpty() && skipTestByDefault())
+            || (!testsToRun.isEmpty() && !testsToRun.contains(thisClassName)))
+        {
+            throw new SkipException("skips-" + thisClassName);
+        }
+    }
+
+    /**
+     * Returns list of string tokens, using ',' delimiter.
+     * If string is null returns empty list.
+     *
+     * @param str the string with values delimited by ','.
+     * @return list of strings.
+     */
+    private List<String> getStringTokens(String str)
+    {
+        if (str != null)
+        {
+            return Stream.of(str.split(","))
+                .filter(e -> e.trim().length() > 0)
+                .collect(Collectors.toList());
+        }
+        else
+        {
+            return Collections.EMPTY_LIST;
+        }
+    }
+
+    /**
+     * Tests can override this to be disabled by default and run only when
+     * explicitly included in the tests to run using
+     * {@link AbstractBaseTest#TESTS_TO_RUN_PNAME} or
+     * {@link AbstractBaseTest#TESTS_TO_INCLUDE_PNAME}.
+     *
+     * @return <code>true</code> to skip current test class
+     * or <code>false</code> otherwise.
+     */
+    public boolean skipTestByDefault()
+    {
+        return false;
+    }
+
+    /**
+     * Wraps assertTrue.
+     *
+     * @param message the message.
+     * @param value the value to check.
+     */
+    public void assertTrue(String message, boolean value)
+    {
+        Assert.assertTrue(value, message);
+    }
+
+    /**
+     * Wraps assertEquals.
+     *
+     * @param obj1 the first object.
+     * @param obj2 the second object.
+     */
+    public void assertEquals(Object obj1, Object obj2)
+    {
+        this.assertEquals(null, obj1, obj2);
+    }
+
+    /**
+     * Wraps assertEquals.
+     *
+     * @param message the message to show.
+     * @param obj1 the first object.
+     * @param obj2 the second object.
+     */
+    public void assertEquals(String message, Object obj1, Object obj2)
+    {
+        Assert.assertEquals(obj1, obj2, message);
+    }
+
+    /**
+     * Wraps assertFalse.
+     * @param message the message to show.
+     * @param value the value to check.
+     */
+    public void assertFalse(String message, boolean value)
```

Comment:
As we've discussed this offline, the purpose of having these methods is to have the other tests ported more easily - with less changes, but seeing those methods here feels weird. There should be at least a comment with an explanation and a plan to be removed as next step (I'm reading commits one by one and don't know yet if this was changed in next commits)

Reply:
Updates, there is a commit porting all assert calls to testng.